src/style.css

```diff
@@ -1 +1,26 @@
 /* add styles here */
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url('images/star.svg');
+  background-position: center;
+  background-repeat: no-repeat;
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
```

Comment:
remove the margin-right in the last star

Reply:
Isn't fixed
You should use `:not(:last-child)` to add margin to all items besides last one
```
.stars__star {
 .......
}

.stars__star:not(:last-child) {
  margin-right: 4px;
}
```